### ISSUE TITLE
Where is the testing set?

### ISSUE THREAD

**Issue by Neo-101** [ZiluM/Deep-learning-for-multi-year-ENSO-Reproduction#4]
In main/Network.ipynb, I see training set, validation set, but no testing set. 

`hist = model.fit(ip_train, nino34_train, validation_data=(ip_var, nino34_var), epochs=20, batch_size=400)`

The trained model is validated on validation set, which means the validation set has been used by the model to adjust the model parameters.

However, to evaluate the performance of model, we need a testing set which has never been used.

So where is the testing set? Can I find it in the code or the processed data?

**Comment by ZiluM** [comment]
I don't set test set.
If you want, you can do that!



At 2022-11-18 11:11:25, "Neo-101" ***@***.***> wrote:

In main/Network.ipynb, I see training set, validation set, but no testing set.

**Comment by Neo-101** [comment]
Got it. Thanks.
